index.js

```diff
@@ -56,14 +58,34 @@ function groups (groups, data, aggregations, thisArg, additionalArgs) {
   data = Array.isArray(data) ? data : data.features
   var args = [undefined, undefined, undefined].concat(additionalArgs)
 
+  var tree = rbush(data.length)
```

Comment:
Good point -- but it still seems intuitively weird to me to have the (max) branching factor equal to `data.length`